### PR TITLE
Update the problem statement, for easier merging

### PR DESCRIPTION
**Pull request by icyflame**
- Same mistakes being done again and again.
- PRs that don't follow the guidelines, should be labelled and
  discarded immediately.

@abinashmeher999 @hargup @kalyan-kumar : Have a look, I feel like we are spending much time writing the same thing in different PR comments.

### CODE REVIEW

**Comment by abinashmeher999** [comment]
I couldn't agree more. I was thinking of adding it in the README. Thanks @icyflame ! :smiley: Maybe add two more points to name the commit message clearly, i.e. it should be evident from commit message itself what was added, and to have a proper name for the PR(`name added` might seem good to beginners but not at all helpful on our part).

**Comment by kalyan-kumar** [comment]
Yes. This should be done immediately.

**Comment by icyflame** [comment]
Done. Two more points added.

Meanwhile, this extension for [Chrome](https://chrome.google.com/webstore/detail/github-canned-responses/lhehmppafakahahobaibfcomknkhoina) and [Firefox](https://addons.mozilla.org/en-US/firefox/addon/github-canned-responses/?src=cb-dl-created) is very useful.

File: README.md [hunk 1]
```diff
@@ -14,7 +14,15 @@ resource](https://help.github.com/articles/good-resources-for-learning-git-and-g
...
 Your entry should maintain alphabetic order and it should be in the format:
 
-`* your name [username](http://github.com/username)`. 
+- `* your name [username](http://github.com/username)`.  
+
+Some more guidelines:
+
+* Your Pull Request must be rebased on the latest master.
+* Your Pull Request must have only one commit (If there are multiple commits, you should squash them.)
+* This commit must have a message that resembles: `Add name to README (Name here)` (Use only first name in case the commit message goes over 70 characters
```
Comment (abinashmeher999) [inline README.md]: Shouldn't this be 50 characters?

Comment (icyflame) [inline README.md]: Oops, yeah! Confused with editor line length.

File: README.md [hunk 2]
```diff
@@ -14,7 +14,15 @@ resource](https://help.github.com/articles/good-resources-for-learning-git-and-g
...
 Your entry should maintain alphabetic order and it should be in the format:
 
-`* your name [username](http://github.com/username)`. 
+- `* your name [username](http://github.com/username)`.  
+
+Some more guidelines:
+
+* Your Pull Request must be rebased on the latest master.
+* Your Pull Request must have only one commit (If there are multiple commits, you should squash them.)
+* This commit must have a message that resembles: `Add name to README (Name here)` (Use only first name in case the commit message goes over 50 characters
```
Comment (abinashmeher999) [inline README.md]: Well :stuck_out_tongue: you missed a parenthesis.

File: README.md [hunk 3]
```diff
@@ -14,7 +14,15 @@ resource](https://help.github.com/articles/good-resources-for-learning-git-and-g
 
 Your entry should maintain alphabetic order and it should be in the format:
 
-`* your name [username](http://github.com/username)`. 
+- `* your name [username](http://github.com/username)`.  
+
+Some more guidelines:
+
+* Your Pull Request must be rebased on the latest master.
+* Your Pull Request must have only one commit (If there are multiple commits, you should squash them.)
```
Comment (abinashmeher999) [inline README.md]: `.` should be outside the `)`.

**Comment by icyflame** [comment]
Updated, please review.

**Comment by abinashmeher999** [comment]
Looks good :+1: Merging.

**Pull request merged**